web/css/joindin.css

```diff
@@ -624,6 +624,19 @@ a.talk.star {
     color: #777;
 }
 
+.event-attendees img {
+    max-width:100%
+}
+
+.event-attendees .row .col {
+    text-align:center;
+}
+
+.event-attendees .row .col .attendeeName {
+    height:40px;
```

Comment:
@Jean85 - this has a fixed height to get round issues with names that wrap onto two lines - keeping the name div consistent to support longer names, without having other issues with broken floats etc, so I think it does need to be height:40px